cypress/integration/tests/login.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import LoginPage from "../../pages";
+// / <reference types="Cypress" />
+
+const loginPage = new LoginPage();
+
+describe("Login", () => {
+  beforeEach(() => {
+    cy.visit("localhost:8080");
```

Comment:
to avoid doing localhost:8080 before every single route there is a baseUrl you can set either in the cypress.json or within the command itself. eventually you would have to account for running your cypress tests against localhost vs. your deployed application with a different URL so you'll most likely have different npm commands to account running with the GUI or in headless mode and against localhost vs. deployed app. your cy.visit would only have the relative route like /bounce_rules or /